src/dotnet/Fable.Client.Browser/Fable.Client.Browser.fsproj

```diff
@@ -30,8 +28,7 @@
     <Compile Include="../Fable.Compiler/FSharp2Fable.Util.fs"/>
     <Compile Include="../Fable.Compiler/FSharp2Fable.fs"/>
     <Compile Include="../Fable.Compiler/Fable2Babel.fs"/>
-
-    <Compile Include="../Fable.Tools/State.fs"/>
+    <Compile Include="../Fable.Compiler/State.fs"/>
```

Comment:
@alfonsogarciacaro Unfortunately no, as we need to compile them with the FABLE_COMPILER flag. This could just be my ignorance though, let me know if there is an easy way to do that.